apps/nowcasting-app/components/map/color-guide-bar.tsx

```diff
@@ -40,13 +40,15 @@ const ColorGuideBar: React.FC<ColorGuideBarProps> = ({ unit }) => {
   }
   return (
     <div className="absolute bg-mapbox-black-700 bottom-12 flex left-0 ml-12 z-20">
-      <div className="flex justify-between text-xs h-full text-ocf-black-600 font-bold relative items-end md:text-sm">
+      <div className="flex justify-between text-xs h-full text-ocf-black-600 font-bold relative items-end md:text-sm 2xl:text-xl 2xl:tracking-wide">
```

Comment:
This is adding to my comment in the review about the size of the `delta-color-guide-bar` in dashboard view. It doesn't seem like you add a `2xl` class in delta view like you do here on line 43.